pkg/kv/iterators.go

```diff
@@ -299,6 +303,9 @@ func (p *PartitionIterator) SeekGE(key []byte) {
 			p.itr.Close() // Close previous before creating new iterator
```

Comment:
above this line - a nil check - need to check the bool

Reply:
Done